MetaGenerator/ExceptionFormatter.cs

```diff
@@ -21,12 +21,22 @@ public ExceptionFormatter(IConsoleWriter consoleWriter, IGlobalOptions globalOpt
 
         public void FormatException(Exception exception)
         {
-            _consoleWriter.WriteLine($"Exception in module: {Module()}");
-            _consoleWriter.WriteLine(exception.Message.Pastel(Color.OrangeRed));
-
-            if (_globalOptions.Verbose && exception.StackTrace is { })
+            if (exception is AggregateException aggregateException)
+            {
+                foreach (var innerException in aggregateException.InnerExceptions)
+                {
+                    FormatException(innerException);
+                }
+            }
+            else
             {
-                _consoleWriter.WriteLine(exception.StackTrace);
+                _consoleWriter.WriteLine($"Exception in module: {Module()}".Pastel(Color.LightGoldenrodYellow));
+                _consoleWriter.WriteLine(exception.Message.Pastel(Color.OrangeRed));
+
+                if (_globalOptions.Verbose && exception.StackTrace is { })
```

Comment:
remove (slight) code duplication by binding `exception.StackTrace` to a variable like so: `exception.StackTrace is { } stackTrace`